### PR TITLE
creating Adder function

### PR DESCRIPTION
**Pull request by pedrocadahia**
# Description
We added a function Adder.  This is due to my calculator is broken, we need to add numbers.

### CODE REVIEW

**Review by EricSchles — CHANGES_REQUESTED** [review]
Overall this functionality looks great!  Exactly what I was hoping for.

I left a few inline comments for what I'd like to see changed.

High level:
* we need to add tests
* documentation
* extend functionality

File: firs_doc.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def adder(a,b):
```
Comment (EricSchles) [inline firs_doc.py:1]: You should add mypy syntax here, so we know what types are allowed.

File: first_doc.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def adder(a,b):
+    return a+b
```
Comment (EricSchles) [inline first_doc.py:29]: It would be great if this was in a class so we could do both infix and prefix notation.

File: firs_doc.py (hunk 1 above)
Comment (EricSchles) [inline firs_doc.py:1]: It would be great if we had a doc string in this function.

**Review by EricSchles — CHANGES_REQUESTED** [review]

**Comment by pedrocadahia** [comment]
> Overall this functionality looks great! Exactly what I was hoping for.
> 
> I left a few inline comments for what I'd like to see changed.
> 
> High level:
> 
> * we need to add tests
> * documentation
> * extend functionality

Hey, I added the changes you requested, please if you need something more, just comment me.

**Review by EricSchles — APPROVED** [review]
Looks good to me!

**Pull request merged**